initCobraToolbox.m

```diff
@@ -171,7 +171,7 @@ function initCobraToolbox()
 %If the Matlab version is prior to Matlab 2016a, we have to use the TC.xml
 %structure to provide autocompletion, otherwise we can use the json
 %definition.
-if MatlabYear < 2015
+if MatlabYear < 2016

```

Comment:
Whats the error you get? 
If its an error during the checks, than its because the function needs user input. 
In general, I don't mind not providing the tab.complete option for earlier versions, but I would prefer if it would be available.